hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -3185,71 +3195,61 @@ private void closeRegionIgnoreErrors(RegionInfo region, final boolean abort) {
    *   If a close was in progress, this new request will be ignored, and an exception thrown.
    * </p>
    *
-   * @param encodedName Region to close
    * @param abort True if we are aborting
+   * @param destination Where the Region is being moved too... maybe null if unknown.
    * @return True if closed a region.
```

Comment:
This method doesn't actually do a close, it merely attempts to schedule the close request for background processing. Thus, "@return {@code true} when the close request is successfully scheduled or the close request is already scheduled, {@code false} otherwise."

Reply:
Let me fix.